priv/apps/itest/test/features/in_flight_exits.feature

```diff
@@ -1,6 +1,17 @@
 Feature: In Flight Exits
   Scenario: Alice starts an In Flight Exit
-    When Alice deposits "1" ETH to the root chain
-    Then Alice should have "1" ETH on the root chain after finality margin
-    When Alice starts an in flight exit
-    Then Alice should have "1" ETH after finality margin
\ No newline at end of file
+    When "Alice" deposits "10" ETH to the root chain
+    When "Bob" deposits "10" ETH to the root chain
+    Then "Alice" should have "10" ETH on the child chain after finality margin
+    Then "Bob" should have "10" ETH on the child chain after finality margin
```

Comment:
Oh, I spotted a simpler thing for the next line, making a separate thread to not hijack the other. It's easy to fix:

`Alice creates a transaction` => `Alice and Bob create a transaction`